packages/editor/src/components/post-template/create-new-template-modal.js

```diff
@@ -86,15 +90,22 @@ export default function CreateNewTemplateModal( { onClose } ) {
 				),
 			] );
 
-		await createTemplate( {
```

Comment:
@talldan Not sure that this is the best way to get the new template id, ie. adding a [return of the new template here](https://github.com/WordPress/gutenberg/pull/57700/files#diff-27537abf1ebe5edf35e1079c28a3bcb39a7122a7c568676394376e1034f37fd6R61). Some of the core actions do return values, eg. `saveEntityRecord`. However, using the `getCurrentTemplateId` doesn't work in this callback as it it set to undefined if run post `createTemplate`. It does return the new template id if run in a timeout here so a case of the store not being updated yet. 

I am sure there is some other easy/obvious way to get the new template id post `createTemplate` but it evades me at the moment - do you have any ideas?